### DISCUSSION TITLE
compound-types/intro

### DISCUSSION THREAD

**Discussion by giscus** [sunface/rust-by-practice#195] [Book Comments]
# compound-types/intro

Learning Rust By Practice, narrowing the gap between beginner and skilled-dev with challenging examples, exercises and projects.

https://zh.practice.rs/compound-types/intro.html

**Comment by mimizao** [comment]
群主，这个界面的下一页按钮好像不生效，我换了其他浏览器也不行。

**Reply by mimizao** [reply]
反馈完就行了，请忽略本条。

**Reply by sunface** [reply]
知道原因了，右边的章节内目录的 HTML 元素遮住了按钮的元素，触发的原因是章节内容太短了，还不够80%屏幕的大小

**Reply by UncleFB** [reply]
我提了一个pr来解决这个问题。但我对前端不是很熟，不知道解决方案是不是足够好~

**Reply by UncleFB** [reply]
https://github.com/sunface/rust-by-practice/pull/389
